app/javascript/components/download/FileDownloadControl.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState, useEffect } from 'react'
+import { fetchBucketFile } from 'lib/scp-api'
+import { bytesToSize } from 'lib/stats'
+
+/** renders a file download control for the given file object */
+export default function FileDownloadControl({ file, bucketName }) {
+  const [remoteImageUrl, setRemoteImageUrl] = useState(null)
+  const fileName = file.upload_file_name
+
+  /** If there is a remote file, try to load it */
+  useEffect(() => {
+    setRemoteImageUrl(null)
+    getBucketLocalUrl(bucketName, fileName).then(setRemoteImageUrl)
+  }, [bucketName, fileName])
+
+  /** get the remote file path */
+  async function getBucketLocalUrl(bucketName, fileName) {
+    const response = await fetchBucketFile(bucketName, fileName)
+    const filefrombucket = await response
+    const respBlob = await filefrombucket.blob()
+    return URL.createObjectURL(respBlob)
+  }
+
+  if (!file.upload_file_name) {
+    return null
+  } else {
+    if (!file.upload_file_name && file.human_data) {
+      return null
+      // TODO once the Sequence Data tab is added update this section for handling external human data files
```

Comment:
Could you note which ticket this TODO corresponds ticket (and ticket the task if isn't already)?  We typically note the ticket beside the TODO, e.g.:

`TODO (SCP-1234): Once the Sequence Data tab...`